src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,144 @@
 package core.basesyntax;
 
+import java.util.LinkedList;
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    static class Node<T> {
+        T value;
+        Node<T> next;
+        Node<T> prev;
+
+        public Node(T value) {
```

Comment:
```suggestion
        public Node(Node<T> prev, T value, Node<T> next) {
```

initialize all properties in constructor